test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -10422,12 +10422,11 @@ def __insert_jobs_in_db(self, jobs, hostname=None):
                          "attr_resource, attr_value, attr_flags)")
 
         job_stmt = ("INSERT INTO pbs.job (ji_jobid, ji_sv_name, ji_state, "
-                    "ji_substate,ji_svrflags, ji_numattr,"
-                    " ji_ordering, ji_priority, ji_stime, ji_endtbdry, "
-                    "ji_queue, ji_destin, ji_un_type, ji_momaddr, "
-                    "ji_momport, ji_exitstat, ji_quetime, ji_rteretry, "
-                    "ji_fromsock, ji_fromaddr, ji_4jid, ji_4ash, "
-                    "ji_credtype, ji_qrank, ji_savetm, ji_creattm)")
+                    "ji_substate, ji_svrflags, ji_stime, "
```

Comment:
This is not working experimental code, see if you can get rid of this also

Reply:
This looks like ptl framework code and I am not well versed with this part of the code.